azurerm/internal/services/hpccache/hpc_cache_resource.go

```diff
@@ -86,6 +86,38 @@ func resourceHPCCache() *schema.Resource {
 				ValidateFunc: validation.IntBetween(576, 1500),
 			},
 
+			"ntp_server": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "time.windows.com",
```

Comment:
does this change depending on the Azure Environment being used? I'm not sure if this is what's used in Azure China/Germany/US Gov?

Reply:
Confirmed with service team that "it appears" the default value is used among all envs.